<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPreferences</class>
 <widget class="QDialog" name="DlgPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabFractal">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeometry">
      <attribute name="title">
       <string>&amp;Geometry</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblX1">
           <property name="text">
            <string>X1: </string>
           </property>
           <property name="buddy">
            <cstring>editX1</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editX1"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lblX2">
           <property name="text">
            <string>X2: </string>
           </property>
           <property name="buddy">
            <cstring>editX2</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="editX2"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblY1">
           <property name="text">
            <string>Y1:</string>
           </property>
           <property name="buddy">
            <cstring>editY1</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editY1"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="lblY2">
           <property name="text">
            <string>Y2: </string>
           </property>
           <property name="buddy">
            <cstring>editY2</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="editY2"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>42</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblIters">
           <property name="text">
            <string>&amp;Iterations:</string>
           </property>
           <property name="buddy">
            <cstring>sbIters</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbIters">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFractal_2">
      <attribute name="title">
       <string>&amp;Fractal</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="rbMandelbrot">
         <property name="text">
          <string>&amp;Mandelbrot set</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>E&amp;xponent: </string>
           </property>
           <property name="buddy">
            <cstring>sbExponent</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbExponent">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>32</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="rbJulia">
         <property name="text">
          <string>&amp;Julia set: </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="llblCRe">
           <property name="text">
            <string>&amp;Real part of c:</string>
           </property>
           <property name="buddy">
            <cstring>editCRe</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editCRe">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblCIm">
           <property name="text">
            <string>&amp;Imaginary part of c: </string>
           </property>
           <property name="buddy">
            <cstring>editCIm</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editCIm">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
